golem_messages/message/tasks.py

```diff
@@ -234,26 +234,41 @@ def __init__(self, *args, **kwargs):
 
 @library.register(TASK_MSG_BASE + 1)
 class WantToComputeTask(ConcentEnabled, base.Message):
+    """
+    A computation Offer.
+
+    A Provider sends it directly to a Requestor as a response to the Requestor's
+    Demand (Task) in order to get work (SubTask ie. TaskToCompute) to do.
+
+    """
     __slots__ = [
-        'node_name',
-        'perf_index',
-        'max_resource_size',
-        'max_memory_size',
-        'num_cores',
-        'price',
-        'concent_enabled',  # Provider notifies requestor
-                            # about his concent status
-
-        'extra_data',       # used to specify additional required
-                            # information about the provider's environment.
-                            # `golem-messages` should be intentionally agnostic
-                            # with regards to the contents of this field.
-
-        'provider_public_key',  # key used for msg signing and encryption
-        'provider_ethereum_public_key',  # used for transactions on blockchain
-        'task_header',  # TaskHeader, should be signed by requestor
+        'node_name',          # Provider's node name (it's a duplicate from
```

Comment:
This comment is not accurate - `node_name` is provider's name while `task_header.owner.node_name` is requestor's name

Reply:
it was suggested by @jiivan, my fault I was not checked it, added fix to #342